src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/ContextTests.cs

```diff
@@ -2,101 +2,126 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
```

Comment:
If possible, it would be nice to have generated .cs files that compare against baseline .cs files (e.g. compare as a big string). This would make it easier to understand what is being generated and we'd be able to detect minor format\whitespace issues if desired.

Reply:
Thanks, I'll approach this in a follow up.

Reply:
> If possible, it would be nice to have generated .cs files that compare against baseline .cs files

+1. We do this in the Logging source generator tests:

https://github.com/dotnet/runtime/blob/84b7c672427b98e70c35c58232ff573625c96f6a/src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Baselines/TestWithDynamicLogLevel.generated.txt#L1-L10